app/scripts/controllers/network/network-controller.test.js

```diff
@@ -192,6 +193,9 @@ class NetworkCommunications {
       customRpcUrl,
     };
     this.networkClientType = networkClientType;
+    if (networkClientType !== 'infura' && networkClientType !== 'custom') {
```

Comment:
[nit] could we use `NetworkClientType.Infura` and `networkClientType.custom`, the enum thats defined in `create-network-client.ts`

Reply:
Oh yeah! Well this check will go away once I convert this file to TypeScript, but I'll definitely make sure to use that enum then.